src/components/Header/Header.tsx

```diff
@@ -0,0 +1,104 @@
+import styles from './Header.module.scss';
+import logo from 'assets/logo.svg';
+import like from 'assets/like.svg';
+import user from 'assets/user.svg';
+import { NavLink } from 'react-router-dom';
+
+const Header = () => {
+  return (
+    <header className={styles.header}>
+      <div className={styles.header__container}>
+        <div className={styles.header__left}>
+          <div className={styles.header__logo}>
+            <NavLink
+              to="/"
+              className={({ isActive, isPending }) =>
+                isPending
+                  ? `${styles.header__logoLink} ${styles.header__logoLinkPending}`
+                  : isActive
+                    ? `${styles.header__logoLink} ${styles.header__logoLinkActive}`
+                    : styles.header__logoLink
+              }
```

Comment:
[+] хороший кейс для использования classnames, ниже в ссылках можно сделать аналогично
```suggestion
              className={({ isActive, isPending }) => classnames(styles.header__logoLink, isPending && styles.header__logoLinkPending, isActive && styles.header__logoLinkActive)
              }
```